static/app/views/dashboardsV2/manage/templateCard.tsx

```diff
@@ -21,16 +20,16 @@ function TemplateCard({title, widgetCount, onPreview, onAdd}: Props) {
       <Header>
         <IconGeneric size="48" />
         <div>
-          {title}
+          {title} {t('Template')}
```

Comment:
maybe we should pass "Template" through in the `title` instead so they are translated together as one unit for localization purposes

Reply:
Good point - I didn't spend any time looking into title. I can make this update